src/components/Header/TopNav/Dropdown.js

```diff
@@ -0,0 +1,190 @@
+import React, { PureComponent } from 'react'
+import styled, { css } from 'styled-components'
+import is from 'styled-is'
+import remcalc from 'remcalc'
+
+import Chevron from '../../Common/Chevron'
+import InnerAnchorItem from './InnerAnchorItem'
+import headerItemStyles from '../headerItemStyles'
+import topNavItemSpacing from './topNavItemSpacing'
+import outlineStyles from '../outlineStyles'
+
+import {
+  fontSizeAndWeight,
+  defaultLightStyles,
+  hoverLightStyles,
+  clickTapLightStyles,
+  activeAndHoverLightStyles,
+  defaultDarkStyles,
+  hoverDarkStyles,
+  clickTapDarkStyles,
+  activeAndHoverDarkStyles
+} from './outerItemStyles'
+
+const DropdownContainer = styled.li`
+  position: relative;
+  cursor: pointer;
+  background: transparent;
+  ${props => {
```

Comment:
when looking at this, I got an idea: we could use [the attr method from styled comps](https://www.styled-components.com/docs/api#attrs), something like

```
const lightStates = {
default: defaultLightStyles,
click: clickTapLightStyles,
hover: hoverLightStyles,
expandedAndHover: activeAndHoverLightStyles
}

const darkStates = {....}

// (We could even export these in a single object, if possible)

const DropdownContainer = styled.li.attrs({
  states: props => (props.themeVariation === 'light' ? lightStates : darkStates),
})`
> span {
  ${props => props.states.default}
}
&:hover > span {
  ${props => props.states.hover}
}
....
`
```

What do you think?